digdag-standards/src/main/java/io/digdag/standards/operator/td/TdOperatorFactory.java

```diff
@@ -533,4 +535,15 @@ else if (c == DELIMITER_CHAR) {
             return escapedValue.toString();
         }
     }
+
+    @VisibleForTesting
+    static StringBuilder createStmtComment(TaskRequest request)
```

Comment:
FYI: Another option may be a method `String wrapStmtWithComment(TaskRequest request, String sql)`

Reply:
That's better 👍  Updated: 83f00c1